course/resources/rqm-ecologized.md

```diff
@@ -0,0 +1,227 @@
+3. Again, link online lectures?
+
+### Week 3
+1. Ecology background knowledge for lab 1
+2. Ecology lab 1 (or similar)
+3. Lab report? Aka send in their code.
+
+### Week 4
+1. Ecology background knowledge for lab 2
+2. Ecology lab 2 (or similar)
+3. Lab report? Aka send in their code.
+
+### Week 5
+1. Introduce data sets, divide intro groups, think about interesting questions to ask, formulate hypothesis/plan of action.
+    - Potentially do this already the week before and have them take a survey with the data they want to work on and their background experience.
+2. Project work, getting started with their own data in R, tidy data principles, how to clean data, setting up a collaborative git environment with Rstudio (how much do we want to get into git?).
+3. Reflections blog post of what they have learnt so far?
+
+### Week 6
+1. Data visualization in general and in R, ggplot, interactive (since plotly is just one additional line)?
+2. Data visualization continued, project specific.
+3. Online tutorial about plotting.
+
+### Week 7
+1. Stuff important for reproducible workflow, metadata, Rmarkdown, Roxygen?
+2. Project work,
+3. Write a data README?
+
+### Week 8-12
+I have left these open for now, since I expect there will be many things that will be reorganized anyways. Ideally one lecture per week from now on would be project work only. Potential ideas:
+    - A couple of additional ecology lessons?
+    - More info about working together on online collaborative platforms? Both for code and writing.
+    - More about data openness?
+    - Guest lectures?
+    - Preparing a scientific item for academic publishing
+    - Licensing and authorship
+- Let me know your overall thoughts on the draft so far. Maybe it is too compressed, I think it is hard to estimate exactly how long we need to spend on certain topics, and I might have been a bit optimistic.
+
+## Labs and lectures from Martin
+These are the topics from what Martin sent us on Dropbox.
+
+### Structured population dynamics
+- Expressing population changes with matrix algebra
+- Learning how to use R for matrix operations and basic plotting
+- Eigenvalues
+- Elasticity of growth rates
+- Gaussian distribution fitting, mean, standard deviation, probability density function.
+
+### Predator prey population dynamics
+- First order, non-linear, differential equations.
+- Stability analysis, non-trivial equilibrium.
+- deSolve package in R.
+
+### Lectures
+I have put a checkmark on the lessons that seems relevant for the couple of labs that Martin forwarded. Note that this is based on "linguistic similarity" rather than me knowing exactly what topics are included in each lecture... I also don't know to what degree these topics are building on eachother, e.g. do students need to have taken all the previous topics to understand predator prey dynamics? At first glance, it seems like this is _not_ the case, but maybe @mbonsma would know better?
```

Comment:
These are all the lectures from the syllabus that was part of one of the documents Martin sent us. I included it for us to get an overview on what was taught in that class, and which lectures are directly relevant to the labs Martin forwarded. Having that said, we don't yet know if Martin wants us to focus on only on what is covered in those couple of labs or if he wants different parts of his class integrated as well. It is, as you said elsewhere, the big wild card that we need to clarify with him on Thursday.